helm_deploy/hmpps-welcome-people-into-prison-api/values.yaml

```diff
@@ -14,6 +14,7 @@ generic-service:
     host: app-hostname.local    # override per environment
     tlsSecretName: hmpps-welcome-to-prison-cert
     path: /
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
```

Comment:
this needs to be inside an annotations tag

Reply:
you're using the `generic-service` - there is no need for this annotation as it is added automatically if you grab the latest version of the chart

Reply:
Good point.
I updated as us suggested. I removed `path: /` as this is a part of the newest version as well